.github/workflows/cypress.yaml

```diff
@@ -0,0 +1,32 @@
+name: End-to-end tests
+
+on:
+    pull_request:
+        branches: [master]
+
+jobs:
+    cypress-run:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout repository
+              uses: actions/checkout@v3
+
+            - name: Use Node.js ${{ matrix.node-version }}
+              uses: actions/setup-node@v3
+              with:
+                  node-version: ${{ matrix.node-version }}
+
+            - name: Install dependencies
+              run: yarn install
+
+            - name: Cypress run
+              uses: cypress-io/github-action@v5
+              with:
+                  build: yarn run build
+                  browser: chrome
+                  start: yarn run serve
+                  record: true
+                  wait-on: 'http://localhost:8081'
+              env:
+                  CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
+                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
do you have a record key?
What's the cypress cloud organization url? It's hard to debug without seeing what actually was ran.

Reply:
I apparently left the key empty when I was setting up the secret. It's working now.

Reply:
https://www.cypress.io/pricing/
There is a limited free tier.
Cypress Cloud is essentially required to do anything with CI/CD because you can't view what happened in the test run.
It looks like you got something working with cloud when running it local.
The fact that the github action run mentions extra tabs error means there is something different going on and we'd need the video recording to get an idea about it.

Disparity between local and CI run environments is very common.